tableaudocumentapi/connection.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, connxml):
         self._dbname = connxml.get('dbname')
         self._server = connxml.get('server')
         self._username = connxml.get('username')
+        self._authentication = connxml.get('authentication')
+        self._class = connxml.get('class')
+
+    def __repr__(self):
```

Comment:
The repr we get by default is `<tableaudocumentapi.connection.Connection object @ abcd1234>`

Ha, oh XML.